app/api/ApplicationApi.js

```diff
@@ -319,9 +319,19 @@ const ApplicationApi = {
                     memo: memo_object
                 });
 
-                return WalletDb.process_transaction(tr, null, true);
+                return WalletDb.process_transaction(tr, null, true).catch(
```

Comment:
I don't mind catching errors per se, but don't think this belongs in this PR. Any reason why it should be included?

Reply:
Hm. I have no idea how this commit get there. I'll update it tomorrow